manifests/app.pp

```diff
@@ -44,7 +44,6 @@
         group   => $gid,
         mode    => '0644',
         content => template($template),
-        require => Package[$::uwsgi::package_name],
-        notify  => Service[$::uwsgi::service_name]
```

Comment:
Yes, that's correct.  UWSGI in Emperor mode detects the modify time of the
vassals and reloads them automatically.

http://uwsgi-docs.readthedocs.org/en/latest/Emperor.html

On Sun, Apr 12, 2015 at 11:30 PM, Josh Smeaton <notifications@github.com>
wrote:

> In manifests/app.pp
> <https://github.com/Engage/puppet-uwsgi/pull/11#discussion_r28213853>:
>
> > @@ -44,7 +44,6 @@
> >          group   => $gid,
> >          mode    => '0644',
> >          content => template($template),
> > -        require => Package[$::uwsgi::package_name],
> > -        notify  => Service[$::uwsgi::service_name]
>
> Is this notify not required? Is the directory monitored by the running
> service?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Engage/puppet-uwsgi/pull/11/files#r28213853>.
>